splunk-otel-android/src/main/java/com/splunk/rum/SplunkRum.java

```diff
@@ -113,6 +117,23 @@ static SplunkRum initialize(SplunkRumBuilder builder, Application application) {
         return INSTANCE;
     }
 
+    public void experimentalSetScreenName(String screenName, String spanType) {
```

Comment:
These two new public API methods needs javadoc so that users can get a better idea about what they do and when they should/shouldn't use these.